app/components/public/session-item.hbs

```diff
@@ -40,7 +40,15 @@
            </div>
            <div class="left floated nine wide column">
              {{#each @session.speakers as |speaker|}}
-                {{speaker.name}} ({{speaker.position}}, {{speaker.organisation}})
+                {{speaker.name}}(
+                  {{#if speaker.position}}
+                      {{speaker.position}}
+                      {{#if speaker.organisation}}               
+                      , {{speaker.organisation}}
+                      {{/if}}
+                    {{else}}
+                   {{speaker.organisation}}
+                  {{/if}})
```

Comment:
Too complex

Reply:
Just if else blocks

Reply:
`[a, b, c].filter(Boolean).join(', ')`

Reply:
use inline if condition with a minimal set of condition and avoid nesting like above.

Reply:
Can you explain what this piece of code does?

Reply:
Joins multiple strings after removing null